frontend/src/pages/dashboard/market-place/components/main-offer/index.tsx

```diff
@@ -144,8 +144,8 @@ const MainOffer: FC<Props> = ({ uid , email}) => {
 						{modalOffer?.shopLocation}
 					</Paragraph>
 					<Paragraph>
-						<Text strong={true}>Doing Offer: </Text>
-						{modalOffer?.doerName}
+						<Text strong={true}>Status: </Text>
```

Comment:
Why did you choose to display status instead? 😅 I thought that all offers' status will be open by default, won't they?